content/en/tracing/trace_retention_and_ingestion/_index.md

```diff
@@ -124,6 +124,8 @@ To configure for 100% ingestion on every service instrumented with a Datadog tra
 DD_TRACE_SAMPLE_RATE=1.0
 ```
 
+**Note:** This may impact your bill if your total ingestion exceeds the included GBs.  For more information, see the [APM Billing][7] page.
```

Comment:
```suggestion
**Note:** This may impact your bill if your total ingestion exceeds the included GBs. For more information, see the [APM Billing][7] page.
```